### PR TITLE
Add Go modules support

### PR DESCRIPTION
**Pull request by ssudake21**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module github.com/ssudake21/tablewriter
```
Comment (mattn) [inline go.mod]: Do not change module name.

Comment (ssudake21) [reply go.mod]: Corrected thanks for pointing out.

**Comment by ssudake21** [comment]
Hi @mattn , Does PR looks good with refreshed commit ?

**Comment by olekukonko** [comment]
@mattn  what do you think about this or should we wait for `go 1.13` ?

**Comment by mattn** [comment]
Thank you

**Pull request merged**